src/Drupal/Commands/pm/PmCommands.php

```diff
@@ -79,7 +80,7 @@ public function enable(array $modules)
         if (!$this->getModuleInstaller()->install($modules, true)) {
             throw new \Exception('Unable to install modules.');
         }
-        if (batch_get()) {
+        if (batch_get() && !$options['skip-translations']) {
```

Comment:
Are you sure there are no other batches, beside translation import? Maybe removing the batch that holds the translation import is a possibility?

Reply:
No, I am not sure. Removing the batch is not possible. I think the chance is small that disabling the batch will cause an actual problem. We have been running Drupal 8 and Drush without batches untill only two weeks ago, when processing batches was introduced. I think there is no other batch function at enabling of modules since no one reported a problem with it.

The alternative is to disable the import function. I have coded it, but it is not a simple fix.

Reply:
@TessaBakker What to do with this remaining point?

Reply:
Adding batches, to support batches and then skipping batches to support an edge case. It doesn't sound great. What can I expect as a module developer, will the "very important" hook_modules_installed() I wrote be supported by Drush or not?